lib/cl/definition-operators.lisp

```diff
@@ -353,7 +353,14 @@
          #+ccl (ccl::record-arglist ',name '(protocol ,@parameter-names))
          (defmethod ,name ((,gprot protocol) ,@(mapcar #'list parameter-names type-names))
            ,@(when documentation `(,documentation))
-           (stream-write-message-begin ,gprot ,identifier 'call
+           (stream-write-message-begin ,gprot
```

Comment:
I don't like the idea of swinging special variable around and depend on dynamic state. we should use different packages for different services.